src/bin/cql-stress-cassandra-stress/java_generate/distribution/fixed.rs

```diff
@@ -58,3 +58,9 @@ impl FixedDistributionFactory {
         );
     }
 }
+
+impl std::fmt::Display for FixedDistributionFactory {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "FIXED({})", self.0)
+    }
+}
```

Comment:
Why do we need to implement Display for factories? The commit message does not explain that.

Reply:
To display the configuration before running the runtime (I'll include it in the commit message).